iks-life-preserver/variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "cluster" { 
+    description = "The cluster ID of the target cluster"
+}
+
+variable "apikey" {
+    description = "An IBM Cloud API Key with permission to invite users to the account"
+}
+
+variable "reference" {
+    description = "A reference tag/string of your choosing (I.e case number, etc.)"
+}
```

Comment:
maybe make this optional by providing a default value? don't feel strongly - fine as is.

Reply:
I had some trouble coming up with a meaningful default. `life-preserver` perhaps? I really expect the customer to provide something here like a case number or something like that. If anyone else feels strongly about it, I'll consider it more.